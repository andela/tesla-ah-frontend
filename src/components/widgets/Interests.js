import React from 'react';

const Interests = (props) => {
  const { content } = props;
  return (
    <div className="col info-box">
      <div className="info-box-content d-flex flex-column">{content}</div>
    </div>
  );
};

export default Interests;
