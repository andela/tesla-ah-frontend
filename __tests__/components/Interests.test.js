/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Interests from '../../src/components/widgets/Interests';

const renderInterests = (args) => {
  const defaultProps = {
    content: '',
  };
  const props = { ...defaultProps, ...args };

  return shallow(<Interests {...props} />);
};

it('renders Interests component', () => {
  const wrapper = renderInterests();
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find('div').length).toBeGreaterThanOrEqual(2);
});
