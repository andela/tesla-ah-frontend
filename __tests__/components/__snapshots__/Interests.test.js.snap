// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Interests component 1`] = `ShallowWrapper {}`;
